packages/quantic/cypress/e2e/default-1/sort/sort.cypress.ts

```diff
@@ -104,6 +154,53 @@ describe('quantic-sort', () => {
           });
       });
 
+      describe('when passing custom options as slots to the component', () => {
+        it('should display the custom options properly', () => {
+          visitSort({useCase: param.useCase}, true, {
+            withCustomSortOptions: true,
+          });
```

Comment:
According to my previous comment
```suggestion
          visitSort({useCase: param.useCase}, {
            withCustomSortOptions: true,
          });
```